libs/core/src/errors/transform.ts

```diff
@@ -0,0 +1,24 @@
+import { ApolloError } from 'apollo-client';
+
+import { DaffErrorCodeMap } from './error-map';
+
+function transformGraphQlError<T extends DaffErrorCodeMap>(
+	error: ApolloError,
+	map: T,
+): Error {
+	if (!error.graphQLErrors.length) return error;
+	const lookup = map[error.graphQLErrors[0].extensions.category];
```

Comment:
I think this prop is platform specific. It may make sense to move this into `drivers` for Magento instead of `core`

Reply:
`driver/src/magento/errors`?

Reply:
moved to driver and added the `daff` prefix to the function name